PWGLF/TableProducer/cascadebuilder.cxx

```diff
@@ -207,48 +204,53 @@ struct cascadeBuilder {
       auto v0 = v0index.v0Data(); //de-reference index to correct v0data in case it exists
 
       std::array<float, 3> pVtx = {v0.collision().posX(), v0.collision().posY(), v0.collision().posZ()};
+
+      auto bachTrackCast = casc.bachelor_as<TCascTracksTo>();
+      auto posTrackCast = v0.posTrack_as<TCascTracksTo>();
+      auto negTrackCast = v0.negTrack_as<TCascTracksTo>();
+
       hCascCandidate->Fill(1.5); //has matched V0
       if (tpcrefit) {
```

Comment:
This probably should be also ignored for Run 3

Reply:
@aalkin thanks for having a look, you're correct! fixing it now

Reply:
Okay, done, this should now be fine, I hope! Considering the importance of the fixes in the cascade side, it would be great if we could merge soon... Thanks!!!